Cargo.toml

```diff
@@ -100,7 +100,7 @@ libp2p-yamux = { version = "0.39.0", path = "muxers/yamux", optional = true }
 multiaddr = { version = "0.14.0" }
 parking_lot = "0.12.0"
 pin-project = "1.0.0"
-rand = "0.7.3" # Explicit dependency to be used in `wasm-bindgen` feature
+rand = "0.8" # Explicit dependency to be used in `wasm-bindgen` feature
```

Comment:
Given the comment, do we still need it now?

Reply:
Compiles without it on my machine.

Reply:
Thanks! Fixed